core/src/test/java/org/commonjava/maven/ext/manip/io/ModelResolverTest.java

```diff
@@ -52,9 +51,7 @@ public void resolveArtifactTest()
                             "cache-dir" ) );
         final GalleyAPIWrapper wrapper = new GalleyAPIWrapper( galleyInfra );
         final ModelIO model = new ModelIO();
-        final Field f = ModelIO.class.getDeclaredField( "galleyWrapper" );
-        f.setAccessible( true );
-        f.set(model, wrapper);
+        FieldUtils.writeField( model, "galleyWrapper", wrapper, true );
 
```

Comment:
Yeah, I think using a library we already have (commons-lang / FieldUtils) is simpler here, since the mocking library will be using this same technique under the covers anyway.